dist/src/main/java/io/camunda/operate/zeebe/OperatePartitionSupplierConfiguration.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.operate.zeebe;
+
+import io.camunda.webapps.zeebe.PartitionSupplier;
+import io.camunda.webapps.zeebe.PartitionSupplierConfigurer;
+import io.camunda.zeebe.broker.Broker;
+import io.camunda.zeebe.client.ZeebeClient;
+import io.camunda.zeebe.gateway.Gateway;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration(proxyBeanMethods = false)
+public class OperatePartitionSupplierConfiguration {
+  @Bean
+  public PartitionSupplier operatePartitionSupplier(
+      @Autowired(required = false) final Broker broker,
+      @Autowired(required = false) final Gateway gateway,
+      @Autowired final ZeebeClient zeebeClient) {
```

Comment:
@houssain-barouni, is a qualifier necessary to get the client from Operate?

Reply:
It is not necessary, as the bean used by Operate is called `zeebeClient` so it takes the name of the method parameter

Reply:
Ah, okay, and in Tasklist, it is named differently. Got it. Thanks.